docs/domains/Domain.md

```diff
@@ -36,6 +37,7 @@ DomainSpec is a description of a domain.
 | `logHome` | string | The in-pod name of the directory in which to store the domain, Node Manager, server logs, server  *.out, and optionally HTTP access log files if `httpAccessLogInLogHome` is true. Ignored if logHomeEnabled is false. |
 | `logHomeEnabled` | Boolean | Specified whether the log home folder is enabled. Not required. Defaults to true if domainHomeSourceType is PersistentVolume; false, otherwise. |
 | `managedServers` | array of [Managed Server](#managed-server) | Configuration for individual Managed Servers. |
+| `maxClusterConcurrentStartup` | number | The maximum number of managed servers that the operator will start in parallel for a cluster, if maxConcurrentStartup is not specified at the cluster level. A default value of 0 means there is no configured limit. |
```

Comment:
managed servers -> Managed Servers
if maxConcurrentStartup is not -> if `maxConcurrentStartup` is not (`maxConcurrentStartup` should be in code font)

Reply:
fixed